src/main.rs

```diff
@@ -210,7 +210,13 @@ fn main() {
                     fg = theme.cmd_failed_fg;
                 }
                 segments.push(Segment::new(bg, fg, root(shell)).dont_escape());
-            }
+            },
+           Module::Newline => {
+                let (mut bg, mut fg) = (theme.username_bg, theme.username_fg);
+                if matches.is_present("newline")  {
+                    segments.push(Segment::new(bg, fg, String::new()).set_newline());
```

Comment:
You're overcomplicating it. You only need  
```
if matches.is_present("newline") {
    println!();
}
```
at the end of the entire function.  
No need to add a new module.

Reply:
Just an FYI, with this guy I couldn't get the newline to pop-up. I swapped it to `println!("\\n");` though and it worked (I'm using bash). I appreciate all the help with this, it was a good learning experience even though you ended up writing the final solution haha.

Reply:
I'm sorry if I took over your fun :(

Reply:
Oh, no way! Sorry if it came across that way, it was a great way to learn and seeing the end solution helped me understand a lot more than when I started.

Reply:
Ah, okay! Awesome!